packages/yourloops/components/header-bars/patient-nav-bar.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2022, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React, { FunctionComponent, memo, MouseEventHandler } from 'react'
+import { Patient } from '../../lib/patient/models/patient.model'
+import Box from '@mui/material/Box'
+import { useAuth } from '../../lib/auth'
+import { makeStyles } from 'tss-react/mui'
+import { Theme } from '@mui/material/styles'
+import { PatientNavBarTabs } from './patient-nav-bar-tabs'
+import { PatientNavBarInfos } from './patient-nav-bar-infos'
+import { PatientNavBarSelect } from './patient-nav-bar-select'
+import ArrowBackIcon from '@mui/icons-material/ArrowBack'
+import { useHistory } from 'react-router-dom'
+
+interface PatientNavBarProps {
+  chartType: string
+  onClickDashboard: MouseEventHandler<HTMLAnchorElement>
+  onClickDaily: MouseEventHandler<HTMLAnchorElement>
+  onSwitchPatient: Function
+  onClickPrint: MouseEventHandler<HTMLButtonElement>
+  onClickTrends: MouseEventHandler<HTMLAnchorElement>
+  patient: Patient
+  patients: Patient[]
```

Comment:
I find it misleading to have 2 properties with a name so similar (`patient` and `patients`), this could be improved in 2 ways:
- Keep `patient` and rename `patients` to `patientList`
- Keep `patients` and rename `patient` to `currentPatient`

This change should also be done in `PatientNavBarSelect` 😄